alpha/apps/kaltura/lib/batch2/kJobsManager.php

```diff
@@ -1976,7 +1976,16 @@ public static function addExportCsvJob(kExportCsvJobData $jobData, $partnerId, $
 	{
 		$batchJob = new BatchJob();
 		$batchJob->setPartnerId($partnerId);
-		
+		$partner = PartnerPeer::retrieveByPK($partnerId);
```

Comment:
I see this logic is repeating in the code so we should probably create something like getPartnerSharedStoargeBaseDir in myCloudUtils and implement the fetching of $storageProfile->getStorageBaseDir() there.